GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Able/RO_Able_Engines.cfg

```diff
@@ -0,0 +1,41 @@
+//	================================================================================
+//	Able Configs
+//	================================================================================
+
+
+// AJ10-37
+@PART[bluedog_Able_Engine]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = AJ10-37
+	@manufacturer = Aerojet
+	@description = Early version of the venerable AJ10 pressure fed upper stage engine using storable propellants. Used on the Able upper stage with the Vanguard and Thor-Able rockets. An upgrade is available for the AJ10-118D as used on Delta B,C,C1 and D.

```

Comment:
These do not have to be set in this node, since they get owerwritten by the `engine_config` patch, and then again in the patch just below here that runs after that.